assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,93 @@
-console.log('***** Music Collection *****')
+console.log("***** Music Collection *****");
+
+let collection = [];
+
+function addToCollection(title, artist, yearPublished) {
+  let newObject = {
+    title: title,
+    artist: artist,
+    yearPublished: yearPublished,
+  };
+  collection.push(newObject);
+  {
+    return newObject;
+  }
```

Comment:
The code block around the `return` is unnecessary, but also won't hurt anything.